.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy Storybook
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  deploy:
+    name: Deploy Storybook
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2-beta
```

Comment:
Do we need to track the `-beta` release branch of this action? I'm not seeing you use anything special here, so I think we can change this to track `@v2` instead of `@v2-beta`.

If you aren't familiar with GitHub Actions, these are just references to Git repos on GitHub itself, with the part after the `@` being any commit hash, branch, or tag name. The `actions` "organization" on GitHub is just where GitHub themselves put official GitHub Actions, and [`setup-node`](https://github.com/actions/setup-node) is just a Git repo. Most of GitHub's actions repos follow a convention where they use semver-ish tag/branch names and where they keep pushing over the tag/branch names that don't specify the full version string. For example, when v2.1.4 was released, they not only pushed a `v2.1.4` tag, but they also pushed over the `v2` tag.

Reply:
`actions/setup-node@v2-beta` is what was being used in the other ymls, so I just left it as-is, but it's an easy change. I changed this to one `v2`, but left the others alone since I'm less familiar with those workflows.